istioctl/pkg/multicluster/remote_secret.go

```diff
@@ -72,6 +68,11 @@ const (
 	DefaultServiceAccountName = "istio-reader-service-account"
 
 	remoteSecretPrefix = "istio-remote-secret-"
+
+	configSecretName = "istio-kubeconfig"
```

Comment:
hmm ... didn't we discuss in another PR that this isn't a very good name? I'd really like to change it to something like `istio-config-secret`. This would be more in the spirit of the naming for the remote secret.

@linsun @stevenctl

Reply:
I would think this is temporary to be compatible with what istio expects for the "main" kubeconfig today.

Reply:
yes, this is for backward compatibility..